### DISCUSSION TITLE
Iota Lottery

### DISCUSSION THREAD

**Discussion by Slowmotion79** [iotaledger/firefly#8061] [Help & Support]
Today i get 2000 two Nft from the Iota Lottery in my Firefly Wallet. 
Is this right or fake?

![image](https://github.com/iotaledger/firefly/assets/148431444/c05a3dcb-828e-410d-ac55-ac014e2e4915)

**Comment by donnalife** [comment]
its FAKE ,, beware they stole my IOTAS, all this morning, I had over 20500 IOTA , worth over 5000 usd that where stolen,
please help!

**Reply by Didel123** [reply]
Meine IOTA-Wallet wurde gestern auch mit dieser Methode gehackt und alle meine IOTAs 21417 wurden gestohlen.

**Comment by donnalife** [comment]
![lot3](https://github.com/iotaledger/firefly/assets/160241647/77e5d1f3-cbdf-4b98-a447-0fd15087ebd7)
![lotery](https://github.com/iotaledger/firefly/assets/160241647/f8a0a645-6a40-4779-8bc4-150a393757e8)
![lotery2](https://github.com/iotaledger/firefly/assets/160241647/a205bf1b-644d-4483-be63-f32da3d531e4)

**Comment by begonaalvarezd** [comment]
Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
In your wallet go to:
1) Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
3) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/bfd0e861-59be-4057-b3e2-588b6f74b027)

**Comment by donnalife** [comment]
IOTA DOEST respond.
Why this lottery appears in a fiable app =?

On Mon, Feb 19, 2024 at 4:54 AM Didel123 ***@***.***> wrote:

> Meine IOTA-Wallet wurde gestern auch mit dieser Methode gehackt und alle
> meine IOTAs 21417 wurden gestohlen.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/iotaledger/firefly/discussions/8061#discussioncomment-8515142>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BGGRP33QJV6WVFPDF54SAADYUMOMHAVCNFSM6AAAAABDL2P34SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKMJVGE2DE>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Rafael Cadosch Delmar
Bogota, Colombia
+57-314-238-8955

**Reply by begonaalvarezd** [reply]
I am part of the IOTA Foundation and I responded warning about the scam, if you read the message I explain "It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps: ..."

I am very sorry that you are a victim but there is nothing we can do about this. The structure of the network is such that this is possible. Anyone can send something like this to you. How many warnings are there all over the place to not enter your seed/phrase into random sites. Using a self custody wallet in crypto is about “being you own bank”, if you give your keys to someone else there is nothing we can do.

**Reply by Didel123** [reply]
Hi,
I'm still waiting for IOTAs for staking for Shimmer and Assembly. What can I do to at least receive them? I can only receive them through the IOTA Wallet. If I set up a new wallet now, I won't be able to receive them, right? Would it be possible to provide me with these IOTAs out of goodwill?"
![Bild_2024-02-19_164045581](https://github.com/iotaledger/firefly/assets/61206216/4aee8a9e-a1a0-4691-b434-bccccb7f11b0)

**Reply by begonaalvarezd** [reply]
You rewards are owned by your wallet addresses, the ones you used to stake originally, we cant do anything with those outputs, only the original mnemonic can unlock those outputs. Those rewards are already received, they are already airdropped, they are just locked, if you are a victim too and still want to be able to have access to your rewards, you can try being faster than the scammer and move them out of the wallet as soon as they unlock

**Reply by Didel123** [reply]
I have changed the Password for the Wallet. May this help to block the scammer ??

**Reply by Didel123** [reply]
The password is needed to send a transaction .. is this true ??

**Reply by begonaalvarezd** [reply]
if you already leaked your stronghold with a password or your mnemnic somewhere else (like the iota lottery scam) changing the password in firefly will not protect you because as mentioned above you do your own custody, the scammer can set his/her own password in a different wallet, with the mnemonic/seed you can do anything

**Reply by begonaalvarezd** [reply]
You dont 100% need the password to sign, what you really need is the seed/mnemonic, in firefly we never expose the mnemonic to the app so that is why we have the password to decrypt the stronghold and be able to sign, but with the mnemonic that's all you need to create a new profile with the funds in firefly or any other wallet, or just use the CLI and sign transactions. With the mnemonic is the hands of a scammer, there is nothing you can do, the mnemonic is the key to your funds, this is how self custody wallets work in crypto (like eg metamask)

**Reply by Didel123** [reply]
so all I can do is to be faster then the scammer ...
